client/packages/invoices/src/StockOut/utils.tsx

```diff
@@ -177,268 +121,6 @@ export const getAllocatedQuantity = (draftStockOutLines: DraftStockOutLine[]) =>
     3
   );
 
-export const issueStock = (
```

Comment:
`allocation` folder should prob get moved here to be shared with prescription, but for now seeing as only used in outbound, kept it there, along with its utils. prescriptions currently has its own copy of these functions